api/server/server.go

```diff
@@ -64,6 +64,49 @@ func writeError(w http.ResponseWriter, status int, err error) {
 	w.Write([]byte(err.Error()))
 }
 
+// RegisterNotAllowedAPIHandler registers a new handler for unpermitted use of API
+func (a *Server) RegisterNotAllowedAPIHandler(handler Handler) {
+	name := handler.Name()
```

Comment:
```
	a.HTTPServer.Router.PathPrefix("/api/" + handler.Name()).HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
		w.WriteHeader(http.StatusMethodNotAllowed)
	})
```

Reply:
Could you please explain the comment? I am sorry, I don't understand :(
Should it be instead of all the described cases?

Reply:
Sorry, my comment was a bit raw :-) Yes, you can use it instead of all the described cases. We simply set a handler for all methods and the URLs with the given prefix

Reply:
OK, will do it. 
Thanks!